spec/services/idv/usps_mail_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe Idv::UspsMail do
+  describe '#mail_spammed?' do
+    context 'when no mail has been sent' do
+      it 'is never spammed' do
+        user = create(:user)
+        subject = Idv::UspsMail.new(user)
+        expect(subject.mail_spammed?).to eq false
+      end
+    end
+
+    context 'when the max pieces of mail have been sent' do
+      it 'is spammed if all the updates have been within the last month' do
+        user = create(:user)
+        subject = Idv::UspsMail.new(user)
+
+        events = Array.new(4) do
+          Event.create(event_type: :usps_mail_sent, user_id: user.id)
+        end
+
+        allow(subject).to receive(:find_user_events).with(:usps_mail_sent, user).
```

Comment:
ditto for the thing below, let's not stub this and actually create objects in the db